### ISSUE TITLE
Add option to disable "are you sure" 'feature' for Replace All in All Opened Documents

### ISSUE THREAD

**Issue by Jay91150** [notepad-plus-plus/notepad-plus-plus#8527]
Replace All in All Opened Documents is, or rather was, a very versatile tool for some of us, but has been hobbled with this new release. Despite what some (ahem [#8432](https://github.com/notepad-plus-plus/notepad-plus-plus/issues/8432)) insinuate, no, people don't delete their active open files and leave themselves with no backups, and then misclick Replace All in All Opened Documents instead of Replace All. (BTW, even if there are people who misclick, the latter would be fixed by giving the Replace All in All Opened Documents function a hotkey like it should have had from the beginning, i.e. Alt+O or Alt+U or something similar. And if you are steadfastly going to refuse to add a hotkey, then don't underline a letter (currently the 'u' in Documents) to imply that there is a hotkey enabled.)

I use this feature in repetition on sets of large files on a routine basis, and I'm far from the only one who does. Not everyone is like [#8432](https://github.com/notepad-plus-plus/notepad-plus-plus/issues/8432)'s OP and we should not be forced to take this new functionality which will slow down workflows drastically. Please add a checkbox or setting that will allow us to disable the 'are you sure' prompt, and use this feature quickly and efficiently as it was meant to be used.

### Steps to Reproduce the Issue
1. Open multiple files
2. Ctrl+H
3. Enter values to find and to replace
4. Mouse over to Replace All in All Opened Documents (because you can't seem to add a hotkey for this function) and click it

### Expected Behavior
All values of 'find' are replaced with 'replace' in all opened documents

### Actual Behavior
Extra popup appears asking me if I'm really sure I meant to click on the command I deliberately clicked on.

### Debug Information
Notepad++ v7.8.8   (32-bit)
Build time : Jun 24 2020 - 00:15:36
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 2004
OS Build : 19041.329
Current ANSI codepage : 1252
Plugins : ComparePlugin.dll DSpellCheck.dll mimeTools.dll NppConverter.dll NppExport.dll

**Comment by sasumner** [comment]
It's not a big deal.  Just confirm the operation.

**Comment by sasumner** [comment]
I still don't think it is a "big deal", but I will reopen this.

There have been a few mentions of it; for example here is another one: 
https://community.notepad-plus-plus.org/topic/19795/deactivate-are-you-sure-you-want-to-replace-all-occurrences-in-all-open-documents

Since (because of #8131) we now have a _Searching_ subpage in the _Preferences_, there's some room to add a "Confirmations" section.  I envision checkbox controls for :

* Confirm interactive replace in all open docs
* Confirm macro replace in all open docs
* Confirm interactive replace in files
* Confirm macro replace in files

The above goes a bit beyond this current issue's desire, but I think it is okay.

**Comment by greentrancer** [comment]
Yes please, I cannot wait for the option to disable "are you sure" popup!

**Comment by sasumner** [comment]
Waiting this one on acceptance/rejection of #8651

**Comment by Yaron10** [comment]
@sasumner,

Using `NativeLangSpeaker::messageBox` instead of `::MessageBox` can be helpful if https://github.com/notepad-plus-plus/notepad-plus-plus/issues/8699 is implemented.

Minor: shouldn't `directory` be double-quoted?

Thank you.

**Comment by sasumner** [comment]
>Using NativeLangSpeaker::messageBox instead of ::MessageBox can be helpful if

👍 I've really never noticed this version of mb before.

>shouldn't directory be double-quoted?

You mean when Replace-in-Files puts up its mbox?
Hmm.  The directory in that box is offset from other text fairly well.
I'd say "It's on its own line" but I haven't seen it with a really long path. :-)
But still, I think it is okay without quoting.
IMO quoting is just visual clutter, and shouldn't be used unless the context demands it.

**Comment by Yaron10** [comment]
@sasumner,

In the `<MessageBox>` section (in the language/localization XML files) the tags contain a `title` and a `message`.

NPP uses `_nativeLangSpeaker.messageBox` in many cases.
For example:
```
int Notepad_plus::doReloadOrNot(const TCHAR *fn, bool dirty)
{
	if (dirty)
		return _nativeLangSpeaker.messageBox("DoReloadOrNotAndLooseChange",
			_pPublicInterface->getHSelf(),
			TEXT("\"$STR_REPLACE$\"\r\rThis file has been modified by another program.\rDo you want to reload it and lose the changes made in Notepad++?"),
			TEXT("Reload"),
			MB_YESNO | MB_APPLMODAL | MB_ICONEXCLAMATION,
			0, // not used
			fn);
	else
		return _nativeLangSpeaker.messageBox("DoReloadOrNot",
			_pPublicInterface->getHSelf(),
			TEXT("\"$STR_REPLACE$\"\r\rThis file has been modified by another program.\rDo you want to reload it?"),
			TEXT("Reload"),
			MB_YESNO | MB_APPLMODAL | MB_ICONQUESTION,
			0, // not used
			fn);
}
```

Sometimes, however, NPP seems to have forgotten the ` _nativeLangSpeaker.messageBox`. :)

The most interesting case for RTL users is the Save message/dialog.
The single MessageBox is LTR whereas the multi Dialog is RTL.
(In `int Notepad_plus::doSaveOrNot(const TCHAR* fn, bool isMulti)`).
***
> You mean when Replace-in-Files puts up its mbox?

Indeed.

> IMO quoting is just visual clutter, and shouldn't be used unless the context demands it.

I tend to agree.
But in most of the messages in the `<MessageBox>` section, the file/folder is double-quoted. :)
***
You always make sure to have strings translatable. :+1: 

I don't want to open an issue for:
```
				generic_string str2display = TEXT("\"");
				str2display += longFileName;
				str2display += TEXT("\" cannot be opened:\nFolder \"");
				str2display += longFileDir;
				str2display += TEXT("\" doesn't exist.");
				::MessageBox(_pPublicInterface->getHSelf(), str2display.c_str(), TEXT("Cannot open file"), MB_OK);
```
(In `NppIO.cpp`).

**Comment by sasumner** [comment]
Changes from this issue released in Notepad++ version 7.9
